hugo/layouts/robots.txt

```diff
@@ -1,4 +1,6 @@
 User-agent: *
 Allow: /
 
+Disallow: /play/*?id=
```

Comment:
This should be `/play/*?*id=` in order to catch `id` no matter where it appears in the query string order, which will be useful as more params are added (or if we already have additional params in URLs).

Reply:
Done